lib/uploader.rb

```diff
@@ -40,9 +40,12 @@ def bag_id
 
   def print_result(qitem_result)
     if qitem_result["status"] == "DONE"
-      pp client.get(qitem_result["bag"])
+      puts "#{external_id} uploaded successfully"
+      true
```

Comment:
I'm thinking we will want to move these `puts` calls over to using Logger. A logger bound to `STDOUT` and available via service locator throughout the app as the way to do output is pretty lightweight and flexible, accommodating levels/timestamps and making it easy to change formatting/destination decisions centrally.

That enhancement doesn't block this work -- we can make a separate issue for it.